tests/cases/fourslash/refactorConvertParamsToDestructuredObject_functionJSDoc.ts

```diff
@@ -0,0 +1,18 @@
+/// <reference path='fourslash.ts' />
+
+/////**
+//// * Return the boolean state of attribute from an element
+//// * /*a*/@param/*b*/ el The source of the attributes.
+//// * @param atty Name of the attribute or a string of candidate attribute names.
+//// * @param def Default boolean value when attribute is undefined.
+//// */
+////export function /*c*/getBoolFromAttribute/*d*/(
+////    /*e*//** inline JSDoc *//*f*/ attr: string | string[],
+////    def: boolean = false): boolean { }

```

Comment:
Does the refactoring here change jsDoc as well? Why not verify new content as well apart from verifying refactor available?

Reply:
So far it does not change JSDoc, that's a future work item.